css/style.css

```diff
@@ -1,7 +1,388 @@
+  border-color: var(--button-active-border-color);
+}
+
+.toggle-menu .active:hover {
+  border-color: #fff;
+}
+
+/* ----------- UI - Side Bar ----------- */
+
+aside {
+  background-color: var(--secondary-color);
+  padding: 15px;
+}
+
+aside h1 {
+  color: var(--tertiary-color);
+}
+
+.logo {
+  max-width: 100%;
+}
+
+aside h2 {
+  margin-top: 20px;
+  color: var(--primary-text-color);
+}
+
+.nav-links {
+  margin-top: 20px;
+  text-align: center;
+}
+
+.nav-links .icon {
+  margin-right: 0;
+}
+
+.graph::before {
+  background-image: url("../images/lightbeam_icon_graph.png");
+}
+
+.list:before {
+  background-image: url("../images/lightbeam_icon_list.png");
+}
+
+.download::before {
+  background-image: url("../images/lightbeam_icon_download.png");
+}
+
+.reset::before {
+  background-image: url("../images/lightbeam_icon_reset.png");
+}
+
+.thumbs-up::before {
+  background-image: url("../images/lightbeam_icon_feedback.png");
+}
+
+/* ----------- UI - Top Bar ----------- */
+
+main {
+  display: grid;
+  grid-template-rows: 10% 90%;
+  background-color: var(--primary-color);
+}
+
+.top-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--tertiary-color);
+  padding: 20px;
+}
+
+.top-bar > div {
+  display: flex;
+}
+
+.top-bar > div > span {
+  margin-right: 15px;
+}
+
+.tracking-protection {
+  display: flex;
+  align-items: center;
+}
+
+.toggle-outer {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 30px;
+  width: 70px;
+  margin-left: 10px;
+  background-color: var(--secondary-color);
+  border-radius: 2px;
+}
+
+.toggle-inner {
+  height: 20px;
+  width: 30px;
+  background-color: var(--primary-color);
+  border-radius: 5px;
+}
+
+.toggle-label {
+  width: 40%;
+  font-size: .75em;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: var(--primary-text-color);
+}
+
+/* ----------- UI - Graph ----------- */
+
+.vis {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 30px;
+}
+
+.vis > div {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.vis h1 {
+  color: var(--primary-text-color);
+}
+
+.vis h2 {
+  color: var(--secondary-text-color);
+}
+
+.info-panel-controls {
+  display: flex;
+  flex-direction: column;
+  margin-right: -30px;
+  margin-top: -30px;
+}
+
+.website {
+  background-color: transparent;
+  border-radius: 3px 0 0 3px;
+  margin-top: 0;
+  box-shadow: none;
+}
+
+.website::before {
+  background-image: url('../images/lightbeam_icon_website.png');
+  margin-right: 0;
+  opacity: .5;
+}
+
+.help {
+  border-radius: 3px 0 0 3px;
+  margin-top: 5px;
+}
+
+.help::before {
+  background-image: url('../images/lightbeam_icon_help.png');
+  margin-right: 0;
+}
+
+.about {
+  border-radius: 3px 0 0 3px;
+  margin-top: 5px;
+}
+
+.about::before {
+  background-image: url('../images/lightbeam_icon_about.png');
+  margin-right: 0;
```

Comment:
`margin: 0;` same as my comment about the other `::before` can this use a different selector so we don't need multiple `margin-right: 0;`. This is super nit picky but just wanted to ask :)